search-books/src/main/java/nix/education/service/SavingBookService.java

```diff
@@ -0,0 +1,53 @@
+package nix.education.service;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import nix.education.entity.Book;
+import nix.education.entity.Tag;
+import nix.education.repository.BookRepository;
+import nix.education.repository.TagRepository;
+import nix.education.request.BookRequest;
+import org.springframework.stereotype.Service;
+
+@Service
+public class SavingBookService {
```

Comment:
Should we implement an interface here? As far as I understand the main intention of this class is to save transactional data. Should we reflect this in the class name? TransactionalDataService or smth like that...

Reply:
Renamed to TransactionalDataBookService